### PR TITLE
Network loss factor postcode

### PR DESCRIPTION
**Pull request by k8mckay**

### CODE REVIEW

File: openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/WH1/certificate_estimation/WH1_PRC_variables.py [hunk 1]
```diff
@@ -138,6 +138,20 @@ def formula(buildings, period, parameters):
...
+    value_type = int
+    entity = Building
+    definition_period = ETERNITY
+    metadata = {
+        "variable-type": "inter-interesting"
+    }
+    def formula(building, period, parameters):
+        postcode = building('WH1_PDRS__postcode', period)
+        network_loss_factor = parameters(period).PDRS.table_network_loss_factor_by_postcode[postcode]
+
```
Comment (alisha17) [inline openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/WH1/certificate_estimation/WH1_PRC_variables.py:151]: postcode = building('WH1_PDRS__postcode', period)
        network_loss_factor = parameters(period).PDRS.table_network_loss_factor_by_postcode
        return zones.calc(postcode)

File: openfisca_nsw_safeguard/tests/ESS_PDRS_Estimator/WH1/WH1_PRC_calculation.yaml [hunk 2]
```diff
@@ -1,210 +1,227 @@
...
+
+- name: test Postcode to DNSP to network loss factor
+  period: 2021
   input:
     WH1_PDRS__postcode:
       [
-        2354,
-        2365,
-        2074
+        2006
```
Comment (alisha17) [inline openfisca_nsw_safeguard/tests/ESS_PDRS_Estimator/WH1/WH1_PRC_calculation.yaml:82]: can we add some more tests?

File: openfisca_nsw_safeguard/tests/ESS_PDRS_Estimator/WH1/WH1_PRC_calculation.yaml [hunk 3]
```diff
@@ -1,210 +1,227 @@
...
-  output:
-    WH1_peak_demand_savings_capacity:
-      [
-        1.538,
-        0.73825
-      ]
-- name: test zones by postcode
-  period: 2022
-  absolute_error_margin: 0
+# - name: test WH1 baseline input power
```
Comment (alisha17) [inline openfisca_nsw_safeguard/tests/ESS_PDRS_Estimator/WH1/WH1_PRC_calculation.yaml:1]: should we uncomment all these tests?

Comment (k8mckay) [reply openfisca_nsw_safeguard/tests/ESS_PDRS_Estimator/WH1/WH1_PRC_calculation.yaml]: This is done and pushed

File: openfisca_nsw_safeguard/tests/ESS_PDRS_Estimator/WH1/WH1_PRC_calculation.yaml (hunk 2 above)
Comment (k8mckay) [reply openfisca_nsw_safeguard/tests/ESS_PDRS_Estimator/WH1/WH1_PRC_calculation.yaml]: Done! All tests passing

**Comment by k8mckay** [comment]
Tests have been cleaned up and conflict resolved. All tests passing on this branch.

**Review by alisha17 — APPROVED** [review]

**Pull request merged**